### ISSUE TITLE
Training code and hardware resources used

### ISSUE THREAD

**Issue by fazliimam** [gengyuanmax/MeVTR#3]
Hi, can I get the training code and know the hardware resources that you used in the paper?

**Comment by gengyuanmax** [comment]
Hi, the training code is [main_task_retrieval_multi.py](https://github.com/gengyuanmax/MeVTR/blob/main/main_task_retrieval_multi.py).

We use 2x48G GPUs for training.

**Comment by Jeonghoon4** [comment]
Hello, can you provide the training scripts, including detailed hyperparameters configurations such as batch_size, max_words, max_frames, etc., for activity and charades datasets?

**Comment by gengyuanmax** [comment]
> Hello, can you provide the training scripts, including detailed hyperparameters configurations such as batch_size, max_words, max_frames, etc., for activity and charades datasets?

The settings is the same as the evaluation scripts and besides, the batch_size is 64 for both and the learning rate is 1e-4.

**Comment by Lucky-Light-Sun** [comment]
Hi, 
what about the trainning time for Charades-Event dataset or ActivityNet Dataset ?

Best wished!
